staff/asier/react-spotify-api-promises/design/scripts/main.js

```diff
@@ -0,0 +1,171 @@
+$(document).ready(function () {
+    "use strict"
+
+    $("#listArtists, #listAlbums, #listSongs, #backToArtists, #error").hide()
+
+
+    const showError = () => {
+        $("#error").empty()
+        $("#error").append('<div class="d-block bg-light p-5"><h1 class="display-1 text-success text-center" >  No artist matching your search! </h1></div>')
+
+    }
+
+    // ------- submit form and get artists, pass them to the api and show them on html -------//
+
+    $('form').submit(function (e) {
+        e.preventDefault();
+        const query = $('input').val()
+
+        $('form').trigger('reset')
+        $("#listArtists").show()
+        $("#listAlbums, #listSongs, #backToArtists, #error").hide()
+
+        spotifyApi.searchArtists(query)
+            .then( listOfArtists => {
+                listResult(listOfArtists)
+            })
+            .catch(err => console.error("err -->", err)) 
+
+    });

```

Comment:
NOTE semicolons are not used lines above, then why would you suddenly used them here and on? to make it uniform, may worth removing them all, and just adding it if - and only if - in some case is necessary, otherwise omit them.